pkg/synthetictests/allowedbackenddisruption/types.go

```diff
@@ -0,0 +1,102 @@
+package allowedbackenddisruption
+
+import (
+	"bytes"
+	_ "embed"
+	"encoding/json"
+	"strconv"
+	"sync"
+)
+
+const (
+	p95ViewQuery = `
```

Comment:
Is this used anywhere? I don't see where, since the results are embedded in the json already

Reply:
> Is this used anywhere? I don't see where, since the results are embedded in the json already

For future us to recalculate p95 and export it as json.